R/class-workbook.R

```diff
@@ -2665,11 +2693,14 @@ wbWorkbook <- R6::R6Class(
             negColour <- "FFFF0000"
           }
 
-          extLst <- self$worksheets[[sheet]]$extLst
```

Comment:
Just FYI: I usually extract objects that are so long to type. It does not really matter in modern R, it is just a pointer to the same memory object, not a copy. especially this `worksheets[[sheet]]$` part always annoyed me. That's why I wanted a shortcut like `$wb(sheet)$`